src/Client.php

```diff
@@ -17,10 +17,12 @@ public function __construct(ClientInterface $httpClient, ResponseBuilder $respon
     }
 
     /**
+     * @param mixed $variables Variables can be in array or MutationObject formats.
+     *
      * @throws \UnexpectedValueException When response body is not a valid json
      * @throws \RuntimeException         When there are transfer errors
      */
-    public function query(string $query, array $variables = null): Response
+    public function query(string $query, $variables = null): Response
```

Comment:
Wouldn't it be better to leave this as "raw query" and use another method for mutation purposes, for example `mutate` or `mutation`?

Reply:
The only difference would be that the mutate(9 can receive also a Mutation object. Is it worth?

Reply:
Well, in my view mutate can *only* receive Mutation object. Right now query and variables expect the "raw" format: it is quite weird to pass half "raw" (query syntax for mutation) and half automatic (mutation object). Ideally the mutation object has the name of the mutation itself, so no raw query is needed. Am I missing anything here?